### ISSUE TITLE
[FEATURE_REQUEST] RSS Feed on dashy main page

### ISSUE THREAD

**Issue by MayhemBill** [Lissy93/dashy#198]
### Is your feature request related to a problem? If so, please describe.

I started looking for dashboards to manage a home lab recently and noticed that not a single one has any sort of feeder on the main page (though most have `bookmark` support) I settled on dashy because I like how it looks and feels. 

### Describe the solution you'd like

Just a way to set an RSS feed on the right side of the page, simple as that.

I set to medium as this is the only thing I am missing from having a "perfect" dashboard application.

### Priority

Medium (Would be very useful)

**Comment by Lissy93** [comment]
Hi @MayhemBill - I'm glad you mentioned this, because one of the next big features I am going to be implementing is widgets, and support for an RSS is top of the list for that. You should see this released before version 2.0 .

In the meantime, will close this issue as it is being tracked on the project board already, but I will update you here once it's implemented.

**Comment by Lissy93** [comment]
Hiya @MayhemBill,
Just to let you know your feature is now merged in #382 :)
Once you've updated to V 1.9.4, you can display a feed with the [rss-widget](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#rss-feed), e,g:

```yaml
sections:
- name: RSS Feed
  icon: fas fa-newspaper
  widgets:
  - type: rss-feed
    options:
      rssUrl: https://cointelegraph.com/editors_pick_rss
```

There are a couple of other pre-built widgets too, which [you can find here](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#rss-feed). This feature is still in beta, so there isn't yet a UI edit form, but that will be coming soon :) 

Hope that helps, and apologies that it took so long, this was a big feature to implement.

---

<p align="center"><i>RSS Example Screenshot:</i><br><img width="700" src="https://i.ibb.co/1r88pvL/rss-feed-example-1.png" /></p>

**Comment by MayhemBill** [comment]
Oh I ended moving to Heimdall, but I might actually move back just I use RSS. Thank you doing this regardless.
